performance_test/helper_scripts/performance_test_file_reader.py

```diff
@@ -1,26 +1,13 @@
 #!/usr/bin/env python
 
 # Copyright 2017 Apex.AI, Inc.
-#
```

Comment:
I think this file should not be changed in this MR at all.

Reply:
This file should actually be removed I think in favor or the new plotter script which automatically determines the number of header lines to read. @jpsamper2009: Can we remove this file in favor of the new plotter script?